config/schedule.rb

```diff
@@ -13,6 +16,10 @@
   rake "order_details:auto_logout"
 end
 
+every 1.minute, roles: [:db] do
+  command "curl #{Rails.application.routes.url_helpers.admin_services_cancel_reservations_for_offline_instruments_url}"
```

Comment:
It has no output on success. Though I just noticed I missed making `curl` do a `POST` instead of a `GET` in the crontab.